### PR TITLE
React - Interactions collection list

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description of change

This is the next step in rewriting the interactions list in React, this PR simply includes the paginated list of interactions without any filters.

## Test instructions

1. View `/interactions/react`

### Before

Screenshot 2021-06-10 at 09 29 54

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/interactions/client/transformers.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    : 'Unknown adviser'
+
+const formatParticipants = (dit_participants) =>
+  dit_participants.length > 1
+    ? 'Multiple advisers'
+    : dit_participants.map((participant) => formatParticipantName(participant))
+
+const getbadgeLabel = (type, hasFeedback = false) => {
+  const badges = {
+    interaction: 'interaction',
```
Comment (ccp92) [inline src/apps/interactions/client/transformers.js:29]: Looking at the screenshots, think this should be a capital Interaction, or else the badge is in lower case.
```suggestion
    interaction: 'Interaction',
```

**Review by ccp92 — APPROVED** [review]
1 minor comment, but otherwise lgtm 👍

File: src/apps/interactions/client/transformers.js (hunk 1 above)
Comment (ian-leggett) [reply src/apps/interactions/client/transformers.js]: Good spot thanks 👍

**Review by paulgain — APPROVED** [review]
LGTM

**Pull request merged**